### PR TITLE
fix(errors inbox): fingerprinting changes

### PR DESCRIPTION
**Pull request by homelessbirds**
Ready for review!

Correct me if I am wrong @elucus but we only have four agents who have documented error fingerprinting with errors inbox:

- Go
- Node
- Python
- Ruby

**Missing:**

- .Net
- Java (pending on 4.13)
- PHP

### CODE REVIEW

**Review by nbaenam — DISMISSED** [review]
It looks good to me!

**Comment by elucus** [comment]
@fallwith I don't think we should have these changes here. Didn't we already update the docs?

File: src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx [hunk 1]
```diff
@@ -92,6 +92,19 @@ To view expected errors, [query your data](/docs/using-new-relic/data/understand
 * To view charts of expected errors, create a query for the [`error.expected`](/docs/insights/insights-data-sources/default-events-attributes/apm-default-event-attributes#transerror-expected) attribute.
 * To create [alert conditions for NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries), use the `error.expected` attribute.
 
+## Error fingerprinting with errors inbox [#error-fingerprinting]
+
+Errors inbox intelligently groups error occurrences to reduce noise and highlight important errors. 
+
+Error events get grouped into an error group when they share the [same fingerprint](/docs/errors-inbox/errors-inbox/#how-groups-work). While our managed rules are able to provide automatic error grouping based on a predefined set of patterns, it is impossible to recognize every possible combination. For this reason, customers can also set their own fingerprint via a callback function if they find that our managed rules are not grouping occurrences accurately. 
+
+To configure a fingerprint, see the agent-specific documentation:
```
Comment (fallwith) [inline src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx:101]: Perhaps "To configure a fingerprint callback," or "To configure custom fingerprint logic," as there is one singular callback but many fingerprints involved.

**Comment by fallwith** [comment]
> @fallwith I don't think we should have these changes here. Didn't we already update the docs?

@elucus We're good here. The only Ruby agent docs specific changes being made here are to create and leverage HTML anchor tags to make it easier to link directly to either the user tracking or the error fingerprinting section. I like the addiitons.

File: src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx:101]: Thanks @fallwith !

**Pull request merged**